SD-K9/src/local-configs/buildings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "hall":{
+    "buildingEntry": "H-806",
+    "floorsEntry":{
+      "6":"H-606",
+      "8":"H-806"
```

Comment:
For building entry, since Hall 1 has been rendered, would it make more sense to put one of the entrances of the ground floor of Hall as the entry point?

Also, please add a floor entry for all the floors that have been rendered!

Reply:
@paulMoose can I do it or are there particular slugs for the entries (like loyola had) ?

Reply:
Joanna is currently working on the vertical transportation, so I think it would be best to let her change that when she is done.